autoload/asyncomplete.vim

```diff
@@ -2,20 +2,20 @@ if !has('timers')
     echohl ErrorMsg
     echomsg 'Vim/Neovim compiled with timers required for asyncomplete.vim.'
     echohl NONE
-    if has('nvim')
-        call asyncomplete#log('neovim compiled with timers required.')
-    else
-        call asyncomplete#log('vim compiled with timers required.')
-    endif
+    call asyncomplete#log('vim/neovim compiled with timers required.')
     finish
 endif
 
 let s:sources = {}
+let s:matches = {}
+let s:already_setup = 0
 let s:change_timer = -1
 let s:last_tick = []
-let s:has_popped_up = 0
-let s:complete_timer_ctx = {}
-let s:already_setup = 0
+let s:startcol = -1
+let s:candidates = []
+let s:script_path = expand('<sfile>:p:h')
+let s:has_lua = has('lua') || has('neovim-0.2.2')
```

Comment:
`has('neovim-0.2.2')` isn't valid. It would need to be `has('nvim-0.2.2')` (see `:help feature-list`)

Reply:
fixed. thanks.